New Map - Endless Arena

culturedcait opened the pull request:
Type: skirmish, scripted
Players: 8p
Size: Giant
Difficulty: very hard

Community link:
https://wildfiregames.com/forum/topic/126722-new-map-endless-arena-skirmish-scripted/

andy5995 reviewed (changes requested):
Pretty cool map!

I get this message in the console repeated many times:

ERROR: CCacheLoader failed to find archived or source file for: "simulation/templates/gaia/fauna_turkey.xml"

And it looks like the map preview needs modifying. https://gitea.wildfiregames.com/0ad/0ad/wiki/Atlas_Manual_Map_Preview

culturedcait commented:
Hi Andy, thanks for review.
I fixed turkeys by replacing them with plain chickens & changed preview to match 0 A.D. convention. It's quite possible turkeys come from different mod I've been using.

Readme.md also updated.

andy5995 commented:
@culturedcait Nice work!
